codebase/config/sync/views.view.export_metadata.yml

```diff
@@ -3475,7 +3475,7 @@ display:
           click_sort_column: uri
           type: link
           settings:
-            trim_length: 80
+            trim_length: 500
```

Comment:
this is arguably outside the scope of the ticket and something I "sneaked" in.  I wanted to change these as I noticed them and not have the overhead of coming back later to fix this separate issue.

Reply:
(this includes the few other spots where I increased the length as well)